### PR TITLE
Update some name

### PR DESCRIPTION
**Pull request by alkabal**
update var probeheight as setterheight
update toolchange-number as toolchange-prep-number
add toolchange-number 
change logic for actual toolnumber self.stat.tool_in_spindle no more used

### CODE REVIEW

File: psng/psng.hal [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-net tool-change            probe.toolchange-change    <=   iocontrol.0.tool-change 
-net tool-changed           probe.toolchange-changed   =>   iocontrol.0.tool-changed
-net tool-prep-number       probe.toolchange-number    <=   iocontrol.0.tool-prep-number
-net tool-prep-loop         iocontrol.0.tool-prepare   <=   iocontrol.0.tool-prepared
+net tool-change            probe.toolchange-change      <=   iocontrol.0.tool-change 
+net tool-changed           probe.toolchange-changed     =>   iocontrol.0.tool-changed
+net tool-number            probe.toolchange-number      <=   iocontrol.0.tool-number
+net tool-prep-number       probe.toolchange-prep-number <=   iocontrol.0.tool-prep-number
+net tool-prep-loop         iocontrol.0.tool-prepare     <=   iocontrol.0.tool-prepared
+net tool-diameter          probe.toolchange-diameter    <=   halui.tool.diameter
```
Comment (kiall) [inline psng/psng.hal:14]: The `probe.toolchange-diameter` pin doesn't exist.

**Review by kiall — CHANGES_REQUESTED** [review]

**Comment by alkabal** [comment]
Oh yes sorry

it’s a potential future pr for diameter measurements (full rewrites)

**Comment by alkabal** [comment]
change done, thank you for pointing that

**Review by kiall — APPROVED** [review]

**Comment by alkabal** [comment]
i speak about ask false to input hal pin ?

**Comment by kiall** [comment]
> i speak about ask false to input hal pin ?

I've no idea what you mean?

**Pull request merged**